dcos/cosmospackage.py

```diff
@@ -296,16 +299,34 @@ def remove_repo(self, name):
         response = self.cosmos_post("repository/delete", params=params)
         return response.json()
 
+    def experimental_package_add(self, dcos_package):
```

Comment:
I would change this to just `package_add`. As we are implementing these commands I would keep in mind that we would like to make it very easy to move them from `dcos experimental ...` to `dcos ...`.